packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -1311,6 +1313,26 @@ abstract class RenderSliver extends RenderObject {
   @override
   void handleEvent(PointerEvent event, SliverHitTestEntry entry) { }
 
+  /// Whether the [SemanticsNode]s associated with this [RenderSliver] should
+  /// be excluded from the semantic scrolling area.
+  ///
+  /// [RenderSliver]s that stay on the screen even though the user has scrolled
+  /// past them (e.g. a pinned app bar) should set this to `true`.
+  set excludeFromSemanticsScrolling(bool value) {
+    if (_excludeFromSemanticsScrolling == value)
+      return;
+    _excludeFromSemanticsScrolling = value;
+    markNeedsSemanticsUpdate();
```

Comment:
Having the semantics annotator always be set also affects the shape of the semantics tree and might cause the creation of additional (empty) semantics nodes. In the current world, if you have nothing to annotate it's better to not offer an annotator.